guide/payments/privacy.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: guide
+title: Transaction privacy
+description: An overview of how to help users maintain their financial privacy while using Bitcoin.
+nav_order: 4
+parent: Payments
+permalink: /guide/payments/privacy/
+main_classes: -no-top-padding
+image: /assets/images/guide/payments/privacy/privacy-preview.jpg
```

Comment:
If I was super picky, I'd complain that this image is 1200x620px and should be 10px taller. Doesn't hugely matter overall, unless you don't want the blue frame in the image to be cropped.

Reply:
oooof. my bad. to be fair my artboard was originally set up as 630 height and then I freehand sized a clipping frame. who wrote that contributing checklist?